public/assets/css/main.css

```diff
@@ -73,8 +73,19 @@ body {
 .service-worker.loading {
   display: block;
 }
+
+.service-worker-status-text {
+  text-align: center;
+}
+
+.service-worker-status {
+  font-size: 1.2em; 
+  border: 1px solid black;
+  padding: 0 3px 0 3px; 
+}
```

Comment:
can you replace `.service-worker-status` with `.service-worker-status-text strong`